subprojects/docs/src/samples/userguide/files/fileTrees/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+import org.gradle.internal.impldep.jcifs.Config
+
+// tag::define[]
+// Create a file tree with a base directory
+var tree: ConfigurableFileTree = fileTree("src/main")
+
+// Add include and exclude patterns to the tree
+tree.include("**/*.java")
+tree.exclude("**/Abstract*")
+
+// Create a tree using path
+tree = fileTree("src").include("**/*.java") as ConfigurableFileTree
```

Comment:
❗️  It should be `fileTree("src") { include("**/*.java") }`
But then it's the same as the next example.

I would simply remove that example from both Groovy and Kotlin samples (I mean `Create a tree using path`). I don't see it's benefit, it's wrongly typed as you noticed, even in Groovy.